.golangci.yaml

```diff
@@ -0,0 +1,22 @@
+version: "2"
+run:
+  concurrency: 4
```

Comment:
Why forcing the concurrency?

Yes, GHA workflows are launched with 4 core, but here you are implying anyone launching golangci-lint locally to use only 4 core. Most developers have more cores.

Reply:
No strong opinion. Happy to do whatever. It's just what I use on my projects because I don't trust the performance of golangci-lint very much. So far v2 seems better, but v1 could be pretty naughty sometimes. Is this a nit or you want it changed?